Source/Microsoft.Teams.Apps.CompanyCommunicator.Common/Repositories/SentNotificationData/SentNotificationDataEntity.cs

```diff
@@ -53,8 +61,43 @@ public class SentNotificationDataEntity : TableEntity
         public int StatusCode { get; set; }
 
         /// <summary>
-        /// Gets or sets the delivery status
+        /// Gets or sets a string representing all of the status code responses for this recipient.
```

Comment:
Can you add a <remarks> section that describes what this really means (reference our chat). The relationship between this, StatusCode, NumberOfAttemptsToSend, and IsStatusCodeFromCreateConversation is unclear.